.github/workflows/check-notice-file.yml

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2022 Robert Bosch GmbH and Microsoft Corporation
```

Comment:
Porbably this workflow should be called "Update notice file" - since the violations should be solved in the PR already.

Reply:
Fixed as discussed.